### PR TITLE
clean up and standardize READMEs

### PR DESCRIPTION
**Pull request by philmillman**
there's a good amount of duplication happening here particularly in the code samples pulled from the docs. We may want to look into automating some of this README generation in the future if it becomes too much of a maintenance burden

### CODE REVIEW

File: packages/docs-site/README.md [hunk 1]
```diff
@@ -29,6 +26,7 @@ All commands are run from the root of the project, from a terminal:
 | `pnpm run preview`         | Preview your build locally, before deploying     |
 | `pnpm run astro ...`       | Run CLI commands like `astro add`, `astro check` |
 | `pnpm run astro -- --help` | Get help using the Astro CLI                     |
+| `cd ../../ && pnpm build:libs`          | Run at root to build the libraries for the docs site            |
```
Comment (theoephraim) [inline packages/docs-site/README.md:29]: can also do `pnpm run -w build:libs` (runs at the workspace level)

**Comment by theoephraim** [comment]
ooo i like these labels - lets wire up the changesets stuff to do it automatically :)

**Comment by philmillman** [comment]
> ooo i like these labels - lets wire up the changesets stuff to do it automatically :)

Sounds good, I'll see what I can do

File: packages/docs-site/README.md (hunk 1 above)
Comment (philmillman) [reply packages/docs-site/README.md:29]:
```suggestion
| `pnpm run -w build:libs`          | Run at root to build the libraries for the docs site            |
```

**Review by theoephraim — APPROVED** [review]

**Pull request merged**